src/main/java/com/powsybl/openloadflow/lf/outerloop/AbstractAreaInterchangeControlOuterLoop.java

```diff
@@ -1,67 +1,62 @@
-/**
- * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- * SPDX-License-Identifier: MPL-2.0
- */
-package com.powsybl.openloadflow.ac.outerloop;
+package com.powsybl.openloadflow.lf.outerloop;
```

Comment:
This file has nolonger copyright notice nor author

Reply:
fixed in [068919d](https://github.com/powsybl/powsybl-open-loadflow/pull/1123/commits/068919d5b58978d567ee7ab4eb96fa8f7b8c7d41)